### DISCUSSION TITLE
What joshuto means?

### DISCUSSION THREAD

**Discussion by azzamsa** [kamiyaa/joshuto#360] [General]
Hi.

I love this project. Would you like to add the origin of "joshuto" as a word? Is is completely a made-up word or ...?

Thank you. :heart:

**Comment by kamiyaa** [comment]
joshu (助手) means assistant in Japanese.
But I didn't like how it rolled off the tongue, so I added -to to the end :joy: 
And according to Google Translate, that means "with assistant"

**Reply by azzamsa** [reply]
Oh, I see :)

Thank you for Joshuto! :heart:
